contracts/contracts-registry/pools/presets/MultiOwnablePoolContractsRegistry.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import {AbstractPoolContractsRegistry} from "../AbstractPoolContractsRegistry.sol";
+import {MultiOwnable} from "../../../access-control/MultiOwnable.sol";
+
+/**
+ * @notice The MultiOwnable preset of PoolContractsRegistry
+ */
+abstract contract MultiOwnablePoolContractsRegistry is
+    AbstractPoolContractsRegistry,
+    MultiOwnable
+{
+    function __MultiOwnablePoolContractsRegistry_init() public initializer {
```

Comment:
It might be better to change the `initializer` to `onlyInitializng`, since you need to inherit from this contract, so other developers might want to extend the `init` function

Reply:
For now it's ok